src/cursor.c

```diff
@@ -0,0 +1,100 @@
+#include "matoya.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+struct MTY_Cursor {
+	MTY_App *app;
+	bool enabled;
+	void *image;
+	uint32_t width;
+	uint32_t height;
+	uint16_t hotX;
+	uint16_t hotY;
```

Comment:
Change to `hot_x` and `hot_y` and update comment regarding metal/dx12.

Reply:
Added comment on `MTY_CursorDraw` in `matoya.h`